run-p2k16-web

```diff
@@ -30,4 +30,5 @@ export STRIPE_PUBLIC_KEY='pk_test_hUoh3MEu4Oa8RjVFwZrFge4b'
 exec env/bin/gunicorn \
     --config "$basedir/infrastructure/gunicorn-config.py" \
     --reload \
+    --bind "0.0.0.0:5000" \
```

Comment:
this seems unrelated to the rest of this commit, and it seems to me that it would make the production p2k16 accessible without going through the normal proxy